Core/Source/MVVM/ScoredModelCollection.swift

```diff
@@ -16,22 +16,22 @@ public final class ScoredModelCollection: ModelCollection, ProxyingCollectionEve
             self?.handleSourceEvent(event)
         }
 
-        updateSections()
+        updateModels()
     }
 
     // MARK: Public
```

Comment:
Totally out of scope for this diff but in retrospect this name `ScoredModelCollection` seems like it doesn't really describe what this MC does, though this is also one of the changes that looks really good in the flattening (sectionLimit -> limit)

Reply:
Agreed — `ScoredModelCollection` is up for rename in https://github.com/dropbox/pilot/issues/54 👍